lib/bandiera/gui.rb

```diff
@@ -64,12 +66,13 @@ def _post_create_group
     # Features
 
     get '/new/feature' do
-      _get_new_feature
-    end
-
-    def _get_new_feature
-      @groups = feature_service.fetch_groups
-
+      group_name = params[:group]
+      if group_name
+        @group = feature_service.find_group(group_name)
+        @back_to_group = true
```

Comment:
quick "hack" go getback to group page after creating feature from a group page.
this creates a hidden field in the feature create form, and is later read by the post handler fn to determine where to return to.
i'm no ruby expert and i'm sure someone can suggest a more clean approach.

Reply:
I can't think of something right now, but this works :)